frontend/packages/sdk/src/utils/signAndSend.ts

```diff
@@ -23,47 +20,35 @@ function callback<TSubmittableResult>(
       }
     }
 
-    (unsub as any)()
+    ;(unsub as any)()
```

Comment:
Weird syntax

Reply:
It might historical reasons: https://stackoverflow.com/questions/7365172/semicolon-before-self-invoking-function

So we have a linter rule for that: https://eslint.org/docs/latest/rules/semi#beforestatementcontinuationchars